webapp/components/Modal/DeleteUserModal.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <ds-modal class="delete-user-modal" :title="title" :is-open="isOpen" @cancel="cancel">
+    <transition name="ds-transition-fade">
+      <ds-flex v-if="success" class="hc-modal-success" centered>
+        <sweetalert-icon icon="success" />
+      </ds-flex>
+    </transition>
+    <div>
+      <ds-section>
+        <ds-flex>
+          <ds-flex-item width="50%">
+            <user-teaser :user="userdata" />
+          </ds-flex-item>
+          <ds-flex-item width="20%">
+            <ds-text size="small">
+              <span class="bold">{{ $t('modals.deleteUser.created') }}</span>
+              <br />
+              <relative-date-time :date-time="userdata.createdAt" />
+            </ds-text>
+          </ds-flex-item>
+          <ds-flex-item width="15%">
+            <ds-text size="small">
+              <span class="bold">{{ $t('common.post', null, userdata.contributionsCount) }}</span>
+              <br />
+              {{ userdata.contributionsCount }}
+            </ds-text>
+          </ds-flex-item>
+          <ds-flex-item width="15%">
+            <ds-text size="small">
+              <span class="bold">{{ $t('common.comment', null, userdata.commentedCount) }}</span>
+              <br />
+              {{ userdata.commentedCount }}
+            </ds-text>
+          </ds-flex-item>
+        </ds-flex>
+      </ds-section>
+    </div>
+
+    <template slot="footer">
+      <base-button class="cancel" @click="cancel">{{ $t('actions.cancel') }}</base-button>
+      <base-button danger filled class="confirm" icon="exclamation-circle" @click="openModal">
+        {{ $t('settings.deleteUserAccount.name') }}
+      </base-button>
+    </template>
+  </ds-modal>
+</template>
+
+<script>
+import gql from 'graphql-tag'
+import { mapMutations } from 'vuex'
+import { SweetalertIcon } from 'vue-sweetalert-icons'
+import RelativeDateTime from '~/components/RelativeDateTime'
+import UserTeaser from '~/components/UserTeaser/UserTeaser'
+
+export default {
+  name: 'DeleteUserModal',
+  components: {
+    UserTeaser,
+    SweetalertIcon,
+    RelativeDateTime,
+  },
+  props: {
+    userdata: { type: Object, required: true },
+  },
+  data() {
+    return {
+      isOpen: true,
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-24T20:31:10Z" title="Tuesday, March 24th 2020, 9:31:10 pm +01:00">Mar 24, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,175 @@
+<template>
+  <ds-modal :title="title" :is-open="isOpen" @cancel="cancel">
+    <transition name="ds-transition-fade">
+      <ds-flex v-if="success" class="hc-modal-success" centered>
+        <sweetalert-icon icon="success" />
+      </ds-flex>
+    </transition>
+    <div>
+      <ds-section>
+        <ds-heading>{{ userdata.name }}</ds-heading>
+        <ds-placeholder>
+          <div>
+            <ds-avatar :name="userdata.name" :image="userdata.avatar" size="x-large" />
+          </div>
+          <br />
+          <ds-text>
+            Slug:
+            <ds-text size="x-large">{{ userdata.slug }}</ds-text>
+          </ds-text>
+          <ds-text>
+            Id:
+            <ds-text size="x-large">{{ userdata.id }}</ds-text>
+          </ds-text>
+          <ds-text>
+            contributionsCount:
+            <ds-text size="x-large">{{ userdata.contributionsCount }}</ds-text>
+          </ds-text>
+          <ds-text>
+            commentedCount:
+            <ds-text size="x-large">{{ userdata.commentedCount }}</ds-text>
+          </ds-text>
+          <ds-text>
+            createdAt:
+            <ds-text size="x-large">{{ userdata.createdAt }}</ds-text>
+          </ds-text>
+        </ds-placeholder>
+      </ds-section>
+    </div>
+
+    <template slot="footer">
+      <base-button class="cancel" @click="cancel">{{ $t('actions.cancel') }}</base-button>
+      <base-button danger filled class="confirm" icon="exclamation-circle" @click="openModal">
+        {{ $t('settings.deleteUserAccount.name') }}
+      </base-button>
+    </template>
+  </ds-modal>
+</template>
+
+<script>
+import gql from 'graphql-tag'
+import { mapMutations } from 'vuex'
+import { SweetalertIcon } from 'vue-sweetalert-icons'
+
+export default {
+  name: 'DeleteUserModal',
+  components: {
+    SweetalertIcon,
+  },
+  props: {
+    userdata: { type: Object, required: true },
+  },
+  data() {
+    return {
+      isOpen: true,
+      success: false,
+      loading: false,
+      isAdmin: this.$store.getters['auth/isAdmin'],
```


Use `mapGetters` helper: https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-28T13:41:12Z" title="Thursday, May 28th 2020, 3:41:12 pm +02:00">May 28, 2020</time>_

---


I agree with @roschaefer here. 👍🏼